.vsts-ci.yml

```diff
@@ -29,7 +29,7 @@ stages:
       enablePublishTestResults: true
       enablePublishBuildAssets: true
       enableTelemetry: true
-      enableSourceBuild: true
+      enableSourceBuild: false
```

Comment:
Why was source-build disabled?  This has caused ripple effects - https://github.com/dotnet/installer/pull/12757#issuecomment-991068628